schedule/yast/multipath.yaml

```diff
@@ -3,15 +3,25 @@ description:    >
   Test installation on machine with virtual multipath hardware.
   Tests successful detection of multipath and installation and
   validate multipath configuration and tools after booting.
+conditional_schedule:
+  multipath_on_full_medium:
+    FLAVOR:
+      'Full-QR':
+        - installation/multipath
+  multipath_on_online_medium:
+    FLAVOR:
+      'Online-QR':
```

Comment:
We run this test for just `Online` flavor, so please add condition for it too. See https://openqa.suse.de/tests/5317550

Reply:
Also, please put a comment explaining this condition at least.

Reply:
I added `Full` and `Online`